include/charge_controller/ChargeController.hpp

```diff
@@ -6,8 +6,32 @@
 #include <charge_controller/dev/BMSManager.hpp>
 #include <charge_controller/dev/LCDDisplay.hpp>
 
-#define MAX_VOLTAGE 12
+//#define MAX_VOLTAGE 12000
+//#define MAX_PACK_VOLTAGE    12000 //in millivolts
+//#define MIN_PACK_VOLTAGE    5000 //in millivolts
+//#define MAX_CELL_VOLTAGE    4200 //in millivolts
+//#define MIN_CELL_VOLTAGE    3000 //in millivolts
+//#define MAX_TEMPERATURE 90.0
+//#define MIN_TEMPERATURE 5.0
+
+//Temp
+#define MAX_PACK_VOLTAGE    12000 //in millivolts
+#define MIN_PACK_VOLTAGE    -1 //in millivolts
+#define MAX_CELL_VOLTAGE    4200 //in millivolts
+#define MIN_CELL_VOLTAGE    -1 //in millivolts
 #define MAX_TEMPERATURE 90.0
+#define MIN_TEMPERATURE -1
+
+#define CHECK_IN_RANGE(V,H,L)  ((L) < (V) && (V) < (H))
```

Comment:
```suggestion
#define IS_IN_RANGE(V,H,L)  ((L) < (V) && (V) < (H))
```
This makes it a bit clearer that this returns a boolean